packages/react/src/misc/adapters/useProfilePriceController.ts

```diff
@@ -0,0 +1,37 @@
+import { formatUnits } from '@ethersproject/units';
+import { UnspecifiedError } from '@lens-protocol/api-bindings';
+import { permissionlessCreator } from '@lens-protocol/blockchain-bindings';
+import {
+  Amount,
+  ChainType,
+  Matic,
+  PromiseResult,
+  assertError,
+  failure,
+  matic,
+  success,
+} from '@lens-protocol/shared-kernel';
+
+import { useSharedDependencies } from '../../shared';
+
+export function useProfilePriceController() {
```

Comment:
I decided to keep all logic in the controller without splitting into gateway/presenter, wdyt?